rs/bitcoin/checker/src/lib.rs

```diff
@@ -26,9 +26,11 @@ pub const INITIAL_MAX_RESPONSE_BYTES: u32 = 4 * 1024;
 /// Retry max response bytes is 400kB
 pub const RETRY_MAX_RESPONSE_BYTES: u32 = 400 * 1024;
 
-pub fn get_tx_cycle_cost(max_response_bytes: u32) -> u128 {
+pub fn get_tx_cycle_cost(max_response_bytes: u32, num_subnet_nodes: u16) -> u128 {
+    let n = num_subnet_nodes as u128;
+    let m = max_response_bytes as u128;
     // 1 KiB for request, max_response_bytes for response
-    49_140_000 + 1024 * 5_200 + 10_400 * (max_response_bytes as u128)
+    (3_000_000 + 60_000 * n) * n + 400 * n * 1024 + 800 * n * m
```

Comment:
could we maybe split this or have some comments regarding which part is what:

1. `base_fee = (3_000_000 + 60_000 * n) * n `
2. `request_fee = 400 * n * 1024`. 
    1. understanding question: Why is the number of bytes in the request always (at most) `1024`?
3. `response_fee=800 * n * m`

Reply:
> Why is the number of bytes in the request always (at most) 1024?

I was following the design doc, which assumes input <= 1024 bytes. Do you prefer using exact bytes?

Reply:
The thing that worries me a bit here, is that it's not only dependent on the request body, but also URL, headers and transform (see [here](https://github.com/dfinity/ic/blob/0e2f787ccdb6d342e2f2a41c7add5a538f04be65/rs/types/types/src/canister_http.rs#L344)). I have the impression this is currently ok, but maybe having a test for this in a separate PR (to avoid slowing down this one) would be a good idea. WDYT?

Reply:
Sounds good. Will add a test for this in another PR.